Libraries/Utilities/ExceptionHelpers.cs

```diff
@@ -0,0 +1,37 @@
+﻿using OrchardCore.ContentManagement;
+using System;
+
+namespace Lombiq.HelpfulLibraries.Libraries.Utilities
+{
+    public static class ExceptionHelpers
+    {
+        public static void ThrowIfNull(object value, string paramName, string message = null)
+        {
+            if (value == default) throw new ArgumentNullException(paramName, message);
+        }
+
+        public static void ThrowIfEmptyOrNull(string value, string paramName, string message = null)
+        {
+            if (string.IsNullOrEmpty(value)) throw new ArgumentNullException(paramName, message);
+        }
+
+        public static void ThrowArgumentException(string paramName = null, string message = null) => 

```

Comment:
Why is this useful? It's just an instantiation.

Reply:
yes

Reply:
![yes](https://i.pinimg.com/originals/45/f9/3c/45f93cc1c6c1e7d77cec6640feac86a2.jpg)